core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -559,6 +578,14 @@ sealed abstract class IO[+A] private () extends IOPlatform[A] {
     else
       flatMap(_ => replicateA_(n - 1))
 
+  /**
+   * Logs the value of this `IO` _(even if it is an error)_ to the stdout.
+   *
+   * This operation is intended as a quick debug, not as proper logging.
+   */
+  def debug: IO[A] =
```

Comment:
```suggestion
  def debug(implicit A: Show[A] = Show.fromToString): IO[A] =
```

Just to be consistent with `IO.println` and to give extra flexibility on the `A` rendering.